sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -418,10 +439,11 @@ def __init__(self,
                id_label,  # type: Optional[str]
                with_attributes,  # type: bool
                timestamp_attribute  # type: Optional[str]
-              ):
+               ):
     self.coder = coders.BytesCoder()
     self.full_topic = topic
     self.id_label = id_label
+    #TODO(BEAM-10869): Remove with_attributes since we will never look at it.
```

Comment:
We can remove `with_attributes` from `PubSubSink` in the cleanup PR. @chamikaramj